pkg/container-utils/docker/docker.go

```diff
@@ -278,3 +266,40 @@ func getContainerImageNamefromImage(image string, containerID string) string {
 	// Case 3 or 4
 	return image
 }
+
+// `getContainerData` takes in two pointers to `dockertypes.ContainerJSON` and `dockertypes.Container`
+// and returns a pointer to a `runtimeclient.ContainerData` and an error. It extracts basic metadata about
+// a Docker container and constructs a `runtimeclient.ContainerData` struct with this information.
+// The function also maps the container state to a runtime client state.
+// If either of the input parameters is `nil`, the function returns an error.
+func getContainerData(containerJSON *dockertypes.ContainerJSON, container *dockertypes.Container) (*runtimeclient.ContainerData, error) {
```

Comment:
As for the previous PR, please extend this function to receive the labels and enrich the ContainerData with the K8s metadata.

Reply:
Thank you for the feedback, Jose :) I added the changes in a new commit, as my first commit was turned into a merge commit and seemed a bit messy to alter that one.